open-metadata-implementation/access-services/data-platform/data-platform-server/src/main/java/org/odpi/openmetadata/accessservices/dataplatform/admin/DataPlatformAdmin.java

```diff
@@ -250,9 +225,8 @@ private OpenMetadataTopicConnector getTopicConnector(Connection topicConnection)
     public void shutdown() {
         try {
             dataPlatformInTopicConnector.disconnect();
-            dataPlatformOutTopicConnector.disconnect();
         } catch (ConnectorCheckedException e) {
-            log.error("Error disconnecting data platform topic connector");
+            log.debug("Error disconnecting data platform topic connector");
```

Comment:
I think error level is more suitable here.

Reply:
Replaced by Audit code with ERROR